packages/config/config/devices/0x014f/smart_retrofit_lighting_kit.json

```
@@ -16,38 +16,15 @@
 	"paramInformation": [
 		{
 			"#": "1",
-			"label": "Dim level when the light bulb is turned On",
-			"description": "Turn Dim Level Memory ON/OFF",
-			"valueSize": 1,
-			"defaultValue": 0,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Disable dim level memory",
-					"value": 0
-				},
-				{
-					"label": "Enable dim level memory",
-					"value": 1
-				}
-			]
+			"$import": "templates/config_template.json#init_brightness"
 		},
 		{
 			"#": "9",
-			"label": "Dim / Bright Step Level",
-			"valueSize": 1,
-			"minValue": 1,
-			"maxValue": 99,
-			"defaultValue": 1
+			"$import": "~/templates/master_template.json#dim_rate"
```

Shouldn't this be the dim step instead?
```suggestion
			"$import": "templates/config_template.json#dim_step_zwave"
```

---

I searched for all instances of this, and only came across these 3 mentioned. The documentation does indeed call it Dim Steps.